cdk/lib/ckan-stack.ts

```diff
@@ -206,5 +206,80 @@ export class CkanStack extends Stack {
       scaleOutCooldown: Duration.seconds(60),
     });
 
+    const ckanCronTaskDefinition = new aws_ecs.FargateTaskDefinition(this, 'ckanTaskDefinition', {
+      cpu: props.cronTaskDef.taskCpu,
+      memoryLimitMiB: props.cronTaskDef.taskMem,
+    });
+    ckanCronTaskDefinition.addToExecutionRolePolicy(new PolicyStatement({
+      effect: Effect.ALLOW,
+      actions: [
+        "kms:Decrypt"
+      ],
+      resources: [
+        secretEncryptionKey.keyArn
+      ]
+    }));
+
+    props.ckanInstanceCredentials.secret!.grantRead(ckanCronTaskDefinition.executionRole!);
+
+    const ckanCronLogGroup = new aws_logs.LogGroup(this, 'ckanCronLogGroup', {
+      logGroupName: `/${props.environment}/registrydata/ckanCron`,
+    });
+
+    const ckanCronContainer = ckanCronTaskDefinition.addContainer('ckanCron', {
+      image: aws_ecs.ContainerImage.fromEcrRepository(ckanRepo, props.envProps.CKAN_IMAGE_TAG),
+      environment: ckanContainerEnv,
+      secrets: ckanContainerSecrets,
+      entryPoint: ['/srv/app/scripts/entrypoint_cron.sh'],
+      logging: aws_ecs.LogDrivers.awsLogs({
+        logGroup: ckanCronLogGroup,
+        streamPrefix: 'ckan-cron-service',
+      }),
+      healthCheck: {
+        command: ['CMD-SHELL', 'curl --fail http://localhost:5000/api/3/action/status_show --user-agent "docker-healthcheck" || exit 1'],
```

Comment:
Health check is wrong as ckan is not actually running in the container, you need to check supervisor instead https://github.com/vrk-kpa/opendata/blob/9777a922195223bf0a26e080a6accef8c6130f31/cdk/lib/ckan-stack.ts#L550